lib/takeaway.rb

```diff
@@ -0,0 +1,34 @@
+require_relative "../lib/basket.rb"
+
+class Takeaway
+
+  attr_reader :menu
+
+  DEFAULT_MENU = {
+    "Burger" => { name: "Burger", price: 3 },       
+    "Chips" => { name: "Chips", price: 1 },
+    "Kebab" => { name: "Kebab", price: 5 } }
+
+  def initialize(menu: DEFAULT_MENU)
+    @menu = menu
+    @basket = Basket.new
```

Comment:
Do you know how to avoid hard coding your Basket class here?  If you can achieve that, you could then use a double for the instance of Basket in your tests.